ci/azure/linux_script

```diff
@@ -65,7 +65,17 @@ make $JOBS install
 cmake .. -DZIG_EXECUTABLE="$(pwd)/release/bin/zig"
 make $JOBS install
 
-release/bin/zig build test -Denable-qemu -Denable-wasmtime
+set +x
+LOG=$(mktemp)
+for step in test-toolchain test-std docs; do
+  echo "* Running step: [$step]"
+  if ! release/bin/zig build $step -Denable-qemu -Denable-wasmtime 2>"$LOG" >&2; then
+    cat "$LOG" >&2
+    exit 1
+  fi
+  echo "  Done."
+done
+set -x
```

Comment:
Here's what I don't like about this:
 * now I have to maintain control flow in bash
 * i hate bash
 * please don't make me maintain bash code
 * now we don't see progress of ongoing tests in the CI
 * if the CI hangs or some other problem happens before the `cat` we miss the output
 * this is not strictly necessary. we could just not do this and it would be fine.

Reply:
The other nice thing about always enabling logging, is you can easily compare the output from success/failure cases.

Reply:
Thanks for your input! I wasn't sure about that change. The logging part has been reverted. Thanks!